addons/mass_mailing/data/mailing_data_templates.xml

```diff
@@ -91,35 +91,37 @@
     <template id="ab_testing_description" name="Mass Mailing: A/B Test Description">
         <div name="ab_testing_description" class="mb-2">
             <t t-if="mailing.ab_testing_completed">
-                <p t-if="mailing.ab_testing_pc == 100">
+                <p t-if="mailing.ab_testing_is_winner_mailing">
                     This <t t-out="mailing.mailing_type_description"/> is the winner of the A/B testing campaign and has been sent to all remaining recipients.
                 </p>
                 <p t-else="">The winner has already been sent. Use <b>Compare Version</b> to get an overview of this A/B testing campaign.</p>
             </t>
             <t t-elif="mailing.ab_testing_mailings_count >= 2">
                 <p>
-                    A sample of <b><t t-out="mailing.ab_testing_pc"/>% of recipients</b> will receive this <b><t t-out="mailing.mailing_type_description"/></b>, and <t t-out="other_ab_testing_pc"/>% receive one of the
-                    <b><t t-out="mailing.ab_testing_mailings_count - 1"/> other versions</b> from the same campaign.
+                    A sample of <b><t t-out="mailing.ab_testing_pc"/>% of recipients</b> will receive this version.<br/>
+                    <t t-if="total_ab_testing_pc > 100 and not mailing.ab_testing_completed and mailing.active">
```

Comment:
I don't think it can be "ab_testing_completed" as we are in a t-elif that had checked "ab_testing_completed" before, does that make sense?

Reply:
Yep, hadn't thought of that